lib/html_sanitize_ex/scrubber/html5.ex

```diff
@@ -704,7 +704,8 @@ defmodule HtmlSanitizeEx.Scrubber.HTML5 do
     "usemap",
     "ismap",
     "width",
-    "height"
+    "height",
+    "size"
```

Comment:
Shouldn't this be `sizes`?

Reply:
yup, i updated, thanks for the feedback 👍